### PR TITLE
CONSOLE-3940: Upgrade dependencies

### PR DESCRIPTION
**Pull request by sg00dwin**
https://issues.redhat.com/browse/CONSOLE-3940

### CODE REVIEW

File: start-console.sh [hunk 1]
```diff
@@ -19,6 +20,7 @@ BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER=$(oc -n openshift-config-managed get co
 set -e
 BRIDGE_K8S_AUTH_BEARER_TOKEN=$(oc whoami --show-token 2>/dev/null)
 BRIDGE_USER_SETTINGS_LOCATION="localstorage"
+BRIDGE_I18N_NAMESPACES="plugin__console-plugin-template"
```
Comment (spadgett) [inline start-console.sh:23]:
```suggestion
BRIDGE_I18N_NAMESPACES="plugin__${npm_package_consolePlugin_name}"
```

**Comment by rhamilto** [comment]
@sg00dwin, this PR doesn't build (which is why CI is failing).  

```
yarn run v1.22.21
$ yarn ts-node node_modules/.bin/webpack serve
$ ts-node -O '{"module":"commonjs"}' node_modules/.bin/webpack serve
package.json (3 errors)
    Console provides shared module react-i18next ^11.7.3 but plugin uses version 13.0.0
    Console provides shared module react-router 5.3.x but plugin uses version 5.2.0
    Console provides shared module react-router-dom 5.3.x but plugin uses version 5.2.0
[webpack-cli] Failed to load '/Users/rhamilto/Git/console-plugin-template/webpack.config.ts' config
[webpack-cli] Error: Validation failed
    at ValidationResult.report (/Users/rhamilto/Git/console-plugin-template/node_modules/src/validation/ValidationResult.ts:75:15)
    at new ConsoleRemotePlugin (/Users/rhamilto/Git/console-plugin-template/node_modules/src/webpack/ConsoleRemotePlugin.ts:285:54)
    at Object.<anonymous> (/Users/rhamilto/Git/console-plugin-template/webpack.config.ts:75:5)
    at Module._compile (node:internal/modules/cjs/loader:1356:14)
    at Module.m._compile (/Users/rhamilto/Git/console-plugin-template/node_modules/ts-node/src/index.ts:1597:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1414:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/rhamilto/Git/console-plugin-template/node_modules/ts-node/src/index.ts:1600:12)
    at Module.load (node:internal/modules/cjs/loader:1197:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1013:12)
    at Module.require (node:internal/modules/cjs/loader:1225:19)
    at require (node:internal/modules/helpers:177:18)
    at WebpackCLI.tryRequireThenImport (/Users/rhamilto/Git/console-plugin-template/node_modules/webpack-cli/lib/webpack-cli.js:244:16)
    at loadConfigByPath (/Users/rhamilto/Git/console-plugin-template/node_modules/webpack-cli/lib/webpack-cli.js:1712:30)
    at WebpackCLI.loadConfig (/Users/rhamilto/Git/console-plugin-template/node_modules/webpack-cli/lib/webpack-cli.js:1832:36)
    at WebpackCLI.createCompiler (/Users/rhamilto/Git/console-plugin-template/node_modules/webpack-cli/lib/webpack-cli.js:2187:29)
    at Command.<anonymous> (/Users/rhamilto/Git/console-plugin-template/node_modules/@webpack-cli/serve/lib/index.js:106:40)
```

**Comment by rhamilto** [comment]
/lgtm
QE Approver:
/assign @yapei
Docs Approver:
/assign @opayne1
PX Approver:
/assign @RickJWagner

**Comment by XiyunZhao** [comment]
This PR has been tested by following the guide in repo, and no issue was found, the example page is worked as expected
/label qe-approved

**Comment by rhamilto** [comment]
@opayne1, mind reviewing and approving?

**Pull request merged**